src/Resources/views/FragmentAdmin/form.html.twig

```diff
@@ -1,9 +1,10 @@
 {% block form %}
     {% set fragmentName = admin.fragmentServices[form.vars.data.type].name|trans({}, 'SonataArticleBundle') %}
     {% set fragmentLabel = 'fragment'|trans({}, 'SonataArticleBundle') %}
+    {% set formdata = {'name': fragmentLabel ~ ' ' ~ fragmentName|escape, 'type' : form.vars.data.backofficeTitle|trans({}, 'SonataArticleBundle') } %}
     <div data-fragment-form="{{ form.vars.id }}"
          data-form-tmp="true"
-         data-formdata='{ "name": "{{ fragmentLabel }} {{ fragmentName|escape }}", "type": "{{ form.vars.data.backofficeTitle|trans({}, 'SonataArticleBundle') }}" }'
+         data-formdata="{{ formdata|json_encode() }}"
```

Comment:
If the extra `formdata` is for clarity, here is another way to get some clarity: use linebreaks.
```suggestion
         data-formdata="{{ {
             'name': fragmentLabel ~ ' ' ~ fragmentName|escape, 
             'type' : form.vars.data.backofficeTitle|trans({}, 'SonataArticleBundle') 
         }|json_encode() }}"
```